docs/userguide/distribution.rst

```diff
@@ -46,7 +46,7 @@ that were built most recently.
 
 If you have to manage automated builds for multiple packages, each with
 different tagging and rotation policies, you may also want to check out the
-`alias`_ command, which would let each package define an alias like ``daily``
+:ref:`alias <alias>` command, which would let each package define an alias like ``daily``
```

Comment:
Honestly, the markup for the commands has to be remade to use the built-in command/option roles but I just used the existing labels for now. Quick and dirty.